ArduCopter/motors.pde

```diff
@@ -68,12 +68,12 @@ static void arm_motors_check()
     }
 }
 
-// auto_disarm_check - disarms the copter if it has been sitting on the ground in manual mode with throttle low for at least 15 seconds
+// auto_disarm_check - disarms the copter if it has been sitting on the ground in manual mode for at least 15 seconds
 // called at 1hz
 static void auto_disarm_check()
 {
     // exit immediately if we are already disarmed or throttle is not zero
-    if (!motors.armed() || !ap.throttle_zero) {
+    if (!motors.armed()) {
```

Comment:
Shouldn't the throttle_zero check be moved to line 82 in conjunction with the mode_has_manual_throttle? Isn't removing this checking making manual modes disarming in mid-air?
I'm new to Ardupilot code, but it seems to me that it would work that way.